src/Svg.php

```diff
@@ -35,8 +36,33 @@ public function contents(): string
         return $this->contents;
     }
 
+    /**
+     * This method adds a title element and an aria-labelledby attribute to the SVG.
+     * To comply with accessibility standards, SVGs should have a title element.
+     * Check accessibility patterns for icons: https://www.deque.com/blog/creating-accessible-svgs/
+     */
+    public function addTitle(): string
+    {
+        // generate a random id for the title element
+        $titleId = 'svg-inline--title-'.Str::random(10);
+
+        // create title element
+        $title = '<title id='.$titleId.'>'.$this->attributes['title'].'</title>';
+
+        // add aria-labelledby attribute to svg element
+        $this->attributes['aria-labelledby'] = $titleId;
+
+        // add title element to svg
+        return preg_replace('/<svg .+?(>)/', "$0 $title", $this->contents);
+    }
+
     public function toHtml(): string
     {
+        // Check if the title attribute is set and add a title element to the SVG
+        if (array_key_exists('title', $this->attributes)) {
+            $this->contents = $this->addTitle();
```

Comment:
One thing I'd change here is pass the title directly to the method as a param.

```suggestion
            $this->contents = $this->addTitle($this->attributes['title']);
```